### PR TITLE
zh-cn: update "fetch" & "abort_static"

### PR DESCRIPTION
**Pull request by Ende93**
### Description
zh-cn: update "fetch" & "abort_static"

### CODE REVIEW

**Comment by Ende93** [comment]
Hi, the `page-type` and `browser-compat` are same with [en-us](https://github.com/mdn/content/blob/73016bea54a98f70dc2aaad0724d0d421ef9839c/files/en-us/web/api/abortsignal/abort_static/index.md?plain=1#L5-L6). Do i need to fix that?

**Comment by yin1999** [comment]
> Hi @yin1999, the `page-type` and `browser-compat` are same with [en-us](https://github.com/mdn/content/blob/73016bea54a98f70dc2aaad0724d0d421ef9839c/files/en-us/web/api/abortsignal/abort_static/index.md?plain=1#L5-L6). Do i need to fix that?

We do not need to preserve those metadata, as the build kit we used would automatically merge the upstream metadata into localized documents. We have [documented this in our translation-guide](https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md#%E5%85%83%E6%95%B0%E6%8D%AE).

**Comment by Ende93** [comment]
Thanks

File: files/zh-cn/web/api/fetch/index.md [hunk 1]
```diff
@@ -23,27 +25,87 @@ Promise<Response> fetch(input[, init]);
...
+
+      - : 布尔值，表示当前用户选择的主题是否应该在关联请求的 {{httpheader("Sec-Browsing-Topics")}} 头中发送。参考[使用 Topics API](/zh-CN/docs/Web/API/Topics_API/Using)。
+
+    - `cache`
+
+      - : 字符串，表示请求如何与浏览器的 [HTTP 缓存](/zh-CN/docs/Web/HTTP/Caching)进行交互。可能的值有 `default`、`no-store`、`reload`、`no-cache`、`force-cache` 和 `only-if-cached`，这些取值在 {{domxref("Request")}} 对象的 {{domxref("Request/cache", "cache")}} 属性的文档中有记录。
+
+    - `credentials`
+
+      - : 控制浏览器如何处理（[cookies](/zh-CN/docs/Web/HTTP/Cookies)、[HTTP authentication](/zh-CN/docs/Web/HTTP/Authentication) 条目和 TLS 客户端凭据）。必须是以下字符串之一：
```
Comment (yin1999) [inline files/zh-cn/web/api/fetch/index.md:53]:
```suggestion
      - : 控制浏览器如何处理凭据（[cookie](/zh-CN/docs/Web/HTTP/Cookies)、[HTTP authentication](/zh-CN/docs/Web/HTTP/Authentication) 条目和 TLS 客户端证书）。必须是以下字符串之一：
```

File: files/zh-cn/web/api/fetch/index.md [hunk 2]
```diff
@@ -23,27 +25,87 @@ Promise<Response> fetch(input[, init]);
...
+
+        - `omit`：告诉浏览器在请求中排除凭据，并忽略响应中发送的任何凭据（例如，任何 {{HTTPHeader("Set-Cookie")}} 头）。
```
Comment (yin1999) [inline files/zh-cn/web/api/fetch/index.md:55]:
```suggestion
        - `omit`：告诉浏览器在请求中排除凭据，并忽略响应中发送的任何凭据（例如，任何 {{HTTPHeader("Set-Cookie")}} 标头）。
```

File: files/zh-cn/web/api/fetch/index.md [hunk 3]
```diff
@@ -23,27 +25,87 @@ Promise<Response> fetch(input[, init]);
...
+        - `same-origin`：告诉浏览器在请求同源 URL 时包含凭据，并使用来自同源 URL 响应中返回的凭据。**这是默认值。**
+        - `include`：告诉浏览器在同源和跨源请求中都包含凭据，并始终使用响应中返回的凭据。
+
+          > **注意：**凭据可以包含在简单和“最终”的跨源请求中，但不应包含在 [CORS 预检请求](/zh-CN/docs/Web/HTTP/CORS#预检请求)中。
```
Comment (yin1999) [inline files/zh-cn/web/api/fetch/index.md:59]:
```suggestion
          > **备注：** 凭据可以包含在简单和“最终”的跨源请求中，但不应包含在 [CORS 预检请求](/zh-CN/docs/Web/HTTP/CORS#预检请求)中。
```

File: files/zh-cn/web/api/fetch/index.md [hunk 4]
```diff
@@ -23,27 +25,87 @@ Promise<Response> fetch(input[, init]);
...
+
+    - `headers`
+
+      - : 任意你想要加到请求上的头部，可以是 {{domxref("Headers")}} 对象或者带有{{jsxref("String", "字符串")}}值的对象字面量。注意[有些头部是被禁止的](/zh-CN/docs/Glossary/Forbidden_header_name)。
```
Comment (yin1999) [inline files/zh-cn/web/api/fetch/index.md:63]:
```suggestion
      - : 任意你想要附加到请求的标头，可以是 {{domxref("Headers")}} 对象或者带有{{jsxref("String", "字符串")}}值的对象字面量。注意[有些标头是被禁止的](/zh-CN/docs/Glossary/Forbidden_header_name)。
```

File: files/zh-cn/web/api/fetch/index.md [hunk 5]
```diff
@@ -23,27 +25,87 @@ Promise<Response> fetch(input[, init]);
...
+
+        > **注意：**请求里可能会有 [`Authorization`](/zh-CN/docs/Web/HTTP/Headers/Authorization) HTTP 头，但是如果请求跨源重定向的话就会被删除。
```
Comment (yin1999) [inline files/zh-cn/web/api/fetch/index.md:65]:
```suggestion
        > **备注：** 请求中可能会有 [`Authorization`](/zh-CN/docs/Web/HTTP/Headers/Authorization) HTTP 标头，但是如果请求跨源重定向的话就会被删除。
```

File: files/zh-cn/web/api/fetch/index.md [hunk 6]
```diff
@@ -23,27 +25,87 @@ Promise<Response> fetch(input[, init]);
...
+
+      - : 包含请求的[子资源完整性](/zh-CN/docs/Web/Security/Subresource_Integrity)值（比如 `sha256-BpfBw7ivV8q2jLiT13fxDYAe2tJllusRSZ273h2nFSE=`）。
+
+    - `keepalive`
+      - : `keepalive` 选项可以允许请求持续保持连接，甚至页面已经关闭的情况。使用 `keepalive` 标志的 Fetch 是 {{domxref("Navigator.sendBeacon()")}} API 的一种替代方案。
+    - `method`
+
+      - : 请求的方法，比如 `"GET"`、`"POST"`，默认值是 `"GET"`。注意 {{HTTPMethod("HEAD")}} 或者 {{HTTPMethod("GET")}} 的 Fetch 请求不会设置 {{httpheader("Origin")}} 头（此行为已在 Firefox 65 中修正——参见 [Firefox bug 1508661](https://bugzil.la/1508661)）。
+        不区分大小写的情况下能匹配上 [RFC 9110](https://www.rfc-editor.org/rfc/rfc9110#name-overview) 中的任意字符串都会自动被转成大小。
+        如果你想使用自定义的方法（比如 `PATCH`），你应该将它变为大写。
```
Comment (yin1999) [inline files/zh-cn/web/api/fetch/index.md:77]:
```suggestion
      - : 请求的方法，比如 `"GET"`、`"POST"`，默认值是 `"GET"`。注意 {{HTTPMethod("HEAD")}} 或者 {{HTTPMethod("GET")}} 的 Fetch 请求不会设置 {{httpheader("Origin")}} 标头（此行为已在 Firefox 65 中修正——参见 [Firefox bug 1508661](https://bugzil.la/1508661)）。不区分大小写的情况下能匹配上 [RFC 9110](https://www.rfc-editor.org/rfc/rfc9110#name-overview) 中的任意字符串都会自动被转成大小。如果你想使用自定义的方法（比如 `PATCH`），你应该将它变为大写。
```

File: files/zh-cn/web/api/fetch/index.md [hunk 7]
```diff
@@ -23,27 +25,87 @@ Promise<Response> fetch(input[, init]);
...
+      - : 指定相对于其他同类型的请求的 fetch 请求的优先级。必须是以下字符串之一：
+        - `high`
+          - : 相对于其他同类型的请求而言，这是一个高优先级的 fetch 请求。
+        - `low`
+          - : 相对于其他同类型的请求而言，这是一个低优先级的 fetch 请求。
+        - `auto`
+          - : 自动确定相对于同类型的其他请求的 fetch 请求的优先级（默认）。
+    - `redirect`
+
+      - : 如何处理**重定向**响应：
```
Comment (yin1999) [inline files/zh-cn/web/api/fetch/index.md:91]:
```suggestion
      - : 如何处理重定向（`redirect`）响应：
```

File: files/zh-cn/web/api/fetch/index.md [hunk 8]
```diff
@@ -23,27 +25,87 @@ Promise<Response> fetch(input[, init]);
...
+
+        - `follow`
+          - : 自动跟随重定向。除非另有说明，否则重定向模式设置为 `follow`。
+        - `error`
+          - : 如果发生重定向，则中止并显示错误。
+        - `manual`
+          - : 调用者打算在另一个上下文中处理响应。详细信息请参阅 [WHATWG fetch standard](https://fetch.spec.whatwg.org/#concept-request-redirect-mode)。
```
Comment (yin1999) [inline files/zh-cn/web/api/fetch/index.md:98]:
```suggestion
          - : 调用者打算在另一个上下文中处理响应。详细信息请参阅 [WHATWG fetch 规范](https://fetch.spec.whatwg.org/#concept-request-redirect-mode)。
```

File: files/zh-cn/web/api/fetch/index.md [hunk 9]
```diff
@@ -23,27 +25,87 @@ Promise<Response> fetch(input[, init]);
...
+
+    - `referrer`
+
+      - : 一个指定请求的引用者的字符串。这可以是同源 URL、`about:client` 或空字符串。
+
+    - `referrerPolicy`
+      - : 指定请求所使用的 [referrer policy](https://w3c.github.io/webappsec-referrer-policy/#referrer-policies)。
+        可能是以下其中之一 `no-referrer`、`no-referrer-when-downgrade`、`same-origin`、`origin`、`strict-origin`、`origin-when-cross-origin`、`strict-origin-when-cross-origin` 或者 `unsafe-url`。
```
Comment (yin1999) [inline files/zh-cn/web/api/fetch/index.md:106]:
```suggestion
      - : 指定请求所使用的 [referrer policy](https://w3c.github.io/webappsec-referrer-policy/#referrer-policies)。可能是以下其中之一 `no-referrer`、`no-referrer-when-downgrade`、`same-origin`、`origin`、`strict-origin`、`origin-when-cross-origin`、`strict-origin-when-cross-origin` 或者 `unsafe-url`。
```

**Review by yin1999 — COMMENTED** [review]
Thanks for the contribution. I've listed some suggestions to take note

**Comment by yin1999** [comment]
Waiting for the upstream PR mdn/content#32765 to be merged :)

**Review by yin1999 — APPROVED** [review]
Thank you @Ende93

**Pull request merged**